ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezSessionState.java

```diff
@@ -310,7 +311,11 @@ protected void openInternal(final HiveConf conf, Collection<String> additionalFi
 
     setupSessionAcls(tezConfig, conf);
 
-    final TezClient session = TezClient.newBuilder("HIVE-" + sessionId, tezConfig)
+    String tezJobName = HiveConf.getVar(conf,ConfVars.HIVETEZJOBNAME); 
+    if (StringUtils.isBlank(tezJobName)){
+      tezJobName = "HIVE-" + sessionId;
```

Comment:
What is the value of setting a hard coded name here?

I think it would be more useful if it would accept a Format String:

`
tezJobName = String.format("HIVE-%s", sessionId);
`

The default value of the configuration should be `HIVE-%s`

Reply:
@belugabehr  This is a good idea and I will do it